js/main.js

```diff
@@ -137,16 +146,62 @@ function editEntry(event) {
         $photoURL.value = data.editing.photoURL;
         $img.setAttribute('src', data.editing.photoURL);
         $notes.value = data.editing.notes;
+        $delete.className = 'delete';
       } else {
         $viewTab[l].className = 'view-tab hidden';
       }
     }
   }
 }
 
+function deleteCheck(event) {
+  $deleteModal.className = 'modal-background';
+}
+
+function cancelDelete(event) {
+  $deleteModal.className = 'modal-background hidden';
+}
+
+function confirmDelete(event) {
+  for (var q = 0; q < data.entries.length; q++) {
+    if (data.entries[q] === data.editing) {
+      data.entries.splice(q, 1);
+    }
+  }
+
+  while ($entryList.firstChild) {
+    $entryList.removeChild($entryList.firstChild);
+  }
+
+  for (var r = 0; r < data.entries.length; r++) {
+    var $entry = createEntryTree(data.entries[r]);
+    $entryList.appendChild($entry);
+  }
```

Comment:
Instead of rebuilding the entire list, you should find the DOM tree for the entry being deleted, and just remove that DOM tree from the DOM.

Reply:
Hey Cody, made the change to remove the one entry instead of rebuilding the entire tree. Demo and snip below!

[delete an entry demo 2.webm](https://user-images.githubusercontent.com/109567499/190458630-08a964d7-1e56-474f-87f3-b844ca21128b.webm)

![image](https://user-images.githubusercontent.com/109567499/190458685-dc32b80c-a576-4959-834c-d46c5ed81518.png)